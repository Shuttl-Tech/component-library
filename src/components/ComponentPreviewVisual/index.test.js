import React from 'react';
import renderer from 'react-test-renderer';

import { ComponentPreviewVisual } from './';

it('ComponentPreviewVisual tree looks like how it should', () => {
	let props = {
		match: {
			params: {
				id: 1
			}
		}
	};

	const component = renderer.create(<ComponentPreviewVisual {...props} />);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});
