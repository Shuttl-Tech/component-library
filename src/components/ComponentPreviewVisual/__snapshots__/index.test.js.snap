// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentPreviewVisual tree looks like how it should 1`] = `
<div
  className="preview"
  id={undefined}
/>
`;
